### PR TITLE
update to lovr 0.15.0

### PR DESCRIPTION
**Pull request by Papaew**

### CODE REVIEW

File: lovr-mouse.lua [hunk 1]
```diff
@@ -107,11 +107,11 @@ end
 
 function mouse.newCursor(source, hotx, hoty)
   if type(source) == 'string' or tostring(source) == 'Blob' then
-    source = lovr.data.newTextureData(source, false)
+    source = lovr.data.newImage(source, false)
   else
     assert(tostring(source) == 'TextureData', 'Bad argument #1 to newCursor (TextureData expected)')
```
Comment (Papaew) [inline lovr-mouse.lua]: And maybe TextureData should be renamed to Image

Comment (bjornbytes) [reply lovr-mouse.lua]: I agree, do you want to make that change here?

Comment (Papaew) [reply lovr-mouse.lua]: Yep

File: lovr-mouse.lua [hunk 2]
```diff
@@ -106,10 +106,10 @@ function mouse.setRelativeMode(enable)
 end
 
 function mouse.newCursor(source, hotx, hoty)
-  if type(source) == 'string' or tostring(source) == 'Blob' then
+  if type(source) == 'string' or tostring(source) == 'Image' then
```
Comment (bjornbytes) [inline lovr-mouse.lua:109]: I think this one ought to remain `Blob` -- `newImage(Image)` doesn't work but `newImage(Blob)` is valid

**Pull request closed without merging**